src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/mercury/InteractiveMapsComponentObject.java

```diff
@@ -127,20 +111,12 @@ public boolean isZoomInButtonEnabled() throws WebDriverException {
     return !zoomInButton.getAttribute("class").contains("disabled");
   }
 
-  public boolean isFilterListScrollable(PerformTouchAction touchAction) {
-    String methodName = "isFilterListScrollable";
-    Shooter shooter = new Shooter();
-    ImageComparison ic = new ImageComparison();
-    File beforeScrolling = shooter.capturePage(driver);
-    clickFilterBox();
-    waitMilliseconds(WAIT_TIME, methodName);
-    touchAction.swipeFromPointToPoint(40, 80, 40, 40, 500, WAIT_TIME);
-    File afterScrolling = shooter.capturePage(driver);
-    return !ic.areFilesTheSame(beforeScrolling, afterScrolling);
-  }
-
   public void switchToMapFrame() {
     waitForElementByElement(lightbox);
     driver.switchTo().frame(mapFrame);
   }
+
+  public void switchToDefaultFrame() {
+    driver.switchTo().defaultContent();
```

Comment:
This method is used just once, and it seems like it's just noise - in my opinion, it's not needed and can be deleted. Is there any reason not to use ```driver.switchTo().defaultContent();``` in place this method is invoked?